RoboFile.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use DigipolisGent\Robo\Symfony\RoboFileBase;
+use DigipolisGent\Robo\Task\Deploy\Ssh\Auth\AbstractAuth;
+
+class RoboFile extends RoboFileBase
+{
+    protected function postSymlinkTask($worker, AbstractAuth $auth, $remote)
+    {
+        $currentProjectRoot = $remote['currentdir'] . '/..';
+        $collection = $this->collectionBuilder();
+        $parent = parent::postSymlinkTask($worker, $auth, $remote);
+        if ($parent) {
+            $collection->addTask($parent);
+        }
+        $collection->taskSsh($worker, $auth)
+            ->remoteDirectory($currentProjectRoot, true)
+            ->exec('composer symfony-scripts');
```

Comment:
Isn't the symlink executed before composer install? Or is it the other way round and is that why this is removed from post-install? In which case, shouldn't we adapt the general symfony flow?